tf2/package.xml

```diff
@@ -22,6 +22,7 @@
   <author>Wim Meeussen</author>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Should I remove this line and add a buildtool_export_depend on ament_cmake_ros here?

Reply:
No, not required.  See my explanation in https://github.com/ros2/rosidl_typesupport_fastrtps/pull/99#issuecomment-1481689789 .  I'll note that it is redundant with `<buildtool_depend>ament_cmake_ros</buildtool_depend>` right below, but since it isn't harmful in any way I think we can just leave it.